include/aikido/constraint/Testable.hpp

```diff
@@ -15,15 +18,21 @@ class Testable
 
   /// Returns true if state satisfies this constraint.
```

Comment:
Document the `TestableOutcome` parameter. Most importantly: (1) what type it must be and (2) what happens if it is not specified. 😉